### PR TITLE
[FEAT] Adding bear to withdraw logic

### PR DESCRIPTION
**Pull request by HBandesh**
# Description

In this PR, I have added all those items which were missing from the withdrawable list. 

Basic bear is not withdrawable if the "Sun Seeker" achievement has not been completed. 

Basic bear not withdrawable:
image

Basic Bear withdrawable:
image

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/types/withdrawables.ts [hunk 1]
```diff
@@ -125,7 +136,25 @@ export const WITHDRAWABLES: Record<InventoryItemName, WithdrawCondition> = {
   "Mysterious Head": true,
   "Golden Bonsai": true,
   "Wicker Man": true,
+  "Engine Core": true,
```
Comment (adamhannigan) [inline src/features/game/types/withdrawables.ts:139]: This was a quest item that is now legacy. We can keep this as `disabled` as there isn't a need to trade it

**Review by adamhannigan — APPROVED** [review]
LGTM. Just the `Engine Core` quest item that is a legacy item and not needed for withdraw 👍

File: src/features/game/types/withdrawables.ts [hunk 2]
```diff
@@ -90,6 +99,7 @@ const globalDefaults = Object.keys(KNOWN_IDS).reduce(
 
 // Group withdraw conditions for common items
 const cropDefaults = buildDefaults(Object.keys(CROPS()), true);
+const fruitDefaults = buildDefaults(Object.keys(FRUIT()), true);
```
Comment (mGranTY) [inline src/features/game/types/withdrawables.ts:102]: Does this make fruits withdrawable?

**Review by adamhannigan — APPROVED** [review]
LGTM 🚀

**Pull request merged**